sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -5,10 +5,113 @@
 ```ts
 
 import { AbortSignalLike } from '@azure/abort-controller';
+import { BaseRequestPolicy } from '@azure/core-http';
+import { HttpMethods } from '@azure/core-http';
+import { HttpOperationResponse } from '@azure/core-http';
+import { OperationArguments } from '@azure/core-http';
+import { OperationSpec } from '@azure/core-http';
+import { RequestPolicy } from '@azure/core-http';
+import { RequestPolicyOptions } from '@azure/core-http';
+import { RestResponse } from '@azure/core-http';
+import { WebResource } from '@azure/core-http';
+
+// @public (undocumented)
+export interface BaseResult extends RestResponse {
+    // (undocumented)
+    _lroData?: LROResponseInfo;
```

Comment:
is this intended to be used by a developer? If not, I'd prefer we use a symbol and keep this internal

Reply:
Nope, this is just for internal use, I think a symbol would make sense